manifests/charts/istiod-remote/templates/mutatingwebhook.yaml

```diff
@@ -16,7 +16,7 @@ webhooks:
   - name: sidecar-injector.istio.io
     clientConfig:
       {{- if .Values.istiodRemote.injectionURL }}
-      url: {{ .Values.istiodRemote.injectionURL }}
+      url: {{ .Values.istiodRemote.injectionURL }}/{{ .Values.global.multiCluster.clusterName }}/{{ .Values.global.network }}
```

Comment:
Can we use query parameters?

Reply:
it is not supported, see https://github.com/istio/istio/issues/23879#issuecomment-630333469

Reply:
Then use /cluster/{{clusterName}}/net/{{network}}

And maybe injectionURL should be 'baseInjectionURL' ? Too late to change I guess. 
Is global.network set in all cases or should it have a | "default" ? 
Same for clusterName.

Reply:
And it would be really great - since we're in effect defining a protocol/API - to have a small something where we document this. Even a clear comment in the implementation - so others don't have to dig trough the code.

Reply:
added a comment and defaults!  good suggestion!

Reply:
agreed might be too late to change to baseInjectionURL